src/evaluation/types/evaluation.types.ts

```
@@ -0,0 +1,41 @@
+import { BaseMessage } from '../../core/types/message.types';
```

ESLint

[error] 'MessageRole' is defined but never used.

(@typescript-eslint/no-unused-vars)